diff --git a/node_modules/@keystatic/core/dist/index-5d64e651.js b/node_modules/@keystatic/core/dist/index-5d64e651.js
index e104981..7ae7f42 100644
--- a/node_modules/@keystatic/core/dist/index-5d64e651.js
+++ b/node_modules/@keystatic/core/dist/index-5d64e651.js
@@ -72,7 +72,7 @@ function RouterProvider(props) {
   if ($[1] === Symbol.for("react.memo_cache_sentinel")) {
     t1 = function navigate(url_0, replace) {
       const newUrl = new URL(url_0, window.location.href);
-      if (newUrl.origin !== window.location.origin || !newUrl.pathname.startsWith("/keystatic")) {
+      if (newUrl.origin !== window.location.origin || !newUrl.pathname.startsWith("/admin")) {
         window.location.assign(newUrl);
         return;
       }
@@ -110,7 +110,7 @@ function RouterProvider(props) {
   let parsedUrl;
   if ($[4] !== url) {
     parsedUrl = new URL(url);
-    const replaced = parsedUrl.pathname.replace(/^\/keystatic\/?/, "");
+    const replaced = parsedUrl.pathname.replace(/^\/admin\/?/, "");
     t4 = replaced === "" ? [] : replaced.split("/").map(decodeURIComponent);
     $[4] = url;
     $[5] = t4;
@@ -5055,7 +5055,7 @@ function CollabAddToPathProvider(props) {
     }
     const avatars = [];
     for (const [clientId, val] of awarenessStates) {
-      if (clientId === yjsInfo.awareness.clientID || !val.user || router.href !== `/keystatic/branch/${val.branch}/${val.location}` || !Array.isArray(val.path) || !areArraysEqual(val.path, props.path)) {
+      if (clientId === yjsInfo.awareness.clientID || !val.user || router.href !== `/admin/branch/${val.branch}/${val.location}` || !Array.isArray(val.path) || !areArraysEqual(val.path, props.path)) {
         continue;
       }
       avatars.push(val.user);
diff --git a/node_modules/@keystatic/core/dist/keystatic-core-ui.js b/node_modules/@keystatic/core/dist/keystatic-core-ui.js
index 2f8809b..877e0f6 100644
--- a/node_modules/@keystatic/core/dist/keystatic-core-ui.js
+++ b/node_modules/@keystatic/core/dist/keystatic-core-ui.js
@@ -1654,7 +1654,7 @@ function PresenceAvatars() {
   if ($[4] !== router.href || $[5] !== t1) {
     t2 = /*#__PURE__*/jsx(HStack, {
       children: t1.map(val => {
-        if (!val.user || router.href !== `/keystatic/branch/${val.branch}/${val.location}`) {
+        if (!val.user || router.href !== `/admin/branch/${val.branch}/${val.location}`) {
           return null;
         }
         return /*#__PURE__*/jsx(Avatar, {
@@ -3142,7 +3142,7 @@ function ItemPageInner(props) {
     t19 = updateResult.kind === "needs-new-branch" && /*#__PURE__*/jsx(CreateBranchDuringUpdateDialog, {
       branchOid: baseCommit,
       onCreate: async newBranch => {
-        const itemBasePath = `/keystatic/branch/${encodeURIComponent(newBranch)}/collection/${encodeURIComponent(collection)}/item/`;
+        const itemBasePath = `/admin/branch/${encodeURIComponent(newBranch)}/collection/${encodeURIComponent(collection)}/item/`;
         router.push(itemBasePath + encodeURIComponent(itemSlug));
         const slug_0 = getSlugFromState(collectionConfig, props.state);
         const hasUpdated_0 = await parentOnUpdate({
@@ -5474,13 +5474,13 @@ function CreateItemInner(props) {
     t25 = createResult.kind === "needs-new-branch" && /*#__PURE__*/jsx(CreateBranchDuringUpdateDialog, {
       branchOid: baseCommit,
       onCreate: async newBranch => {
-        router.push(`/keystatic/branch/${encodeURIComponent(newBranch)}/collection/${encodeURIComponent(props.collection)}/create`);
+        router.push(`/admin/branch/${encodeURIComponent(newBranch)}/collection/${encodeURIComponent(props.collection)}/create`);
         if (await props.createItem({
           branch: newBranch,
           sha: baseCommit
         })) {
           const slug_0 = getSlugFromState(collectionConfig, props.state);
-          router.push(`/keystatic/branch/${encodeURIComponent(newBranch)}/collection/${encodeURIComponent(props.collection)}/item/${encodeURIComponent(slug_0)}`);
+          router.push(`/admin/branch/${encodeURIComponent(newBranch)}/collection/${encodeURIComponent(props.collection)}/item/${encodeURIComponent(slug_0)}`);
         }
       },
       reason: createResult.reason,
@@ -6800,7 +6800,7 @@ function SingletonPageInner(props) {
     t28 = props.updateResult.kind === "needs-new-branch" && /*#__PURE__*/jsx(CreateBranchDuringUpdateDialog, {
       branchOid: baseCommit,
       onCreate: async newBranch => {
-        router.push(`/keystatic/branch/${encodeURIComponent(newBranch)}/singleton/${encodeURIComponent(props.singleton)}`);
+        router.push(`/admin/branch/${encodeURIComponent(newBranch)}/singleton/${encodeURIComponent(props.singleton)}`);
         props.onUpdate({
           branch: newBranch,
           sha: baseCommit
@@ -7907,7 +7907,7 @@ function KeystaticCloudAuthCallback({
           project,
           validUntil: Date.now() + parsed.expires_in * 1000
         }));
-        router.push(`/keystatic/${storedState_0.from}`);
+        router.push(`/admin/${storedState_0.from}`);
       })().catch(error_0 => {
         setError(error_0);
       });
@@ -7999,7 +7999,7 @@ function RedirectToBranch(props) {
         }
       }
       if (data !== null && data !== void 0 && (_data$repository = data.repository) !== null && _data$repository !== void 0 && _data$repository.defaultBranchRef) {
-        push(`/keystatic/branch/${encodeURIComponent(data.repository.defaultBranchRef.name)}`);
+        push(`/admin/branch/${encodeURIComponent(data.repository.defaultBranchRef.name)}`);
       }
       if (props.config.storage.kind === "github" && !(data !== null && data !== void 0 && (_data$repository2 = data.repository) !== null && _data$repository2 !== void 0 && _data$repository2.id) && (error === null || error === void 0 || (_error$graphQLErrors = error.graphQLErrors) === null || _error$graphQLErrors === void 0 || (_error$graphQLErrors = _error$graphQLErrors[0]) === null || _error$graphQLErrors === void 0 || (_error$graphQLErrors = _error$graphQLErrors.originalError) === null || _error$graphQLErrors === void 0 ? void 0 : _error$graphQLErrors.type) === "NOT_FOUND" || (error === null || error === void 0 || (_error$graphQLErrors2 = error.graphQLErrors) === null || _error$graphQLErrors2 === void 0 || (_error$graphQLErrors2 = _error$graphQLErrors2[0]) === null || _error$graphQLErrors2 === void 0 || (_error$graphQLErrors2 = _error$graphQLErrors2.originalError) === null || _error$graphQLErrors2 === void 0 ? void 0 : _error$graphQLErrors2.type) === "FORBIDDEN") {
         window.location.href = "/api/keystatic/github/repo-not-found";
@@ -8160,7 +8160,7 @@ function PageInner(t0) {
       return t3;
     }
     branch = params[1];
-    basePath = `/keystatic/branch/${encodeURIComponent(branch)}`;
+    basePath = `/admin/branch/${encodeURIComponent(branch)}`;
     let t3;
     if ($[20] !== params) {
       t3 = parseParamsWithoutBranch(params.slice(2));
@@ -8180,7 +8180,7 @@ function PageInner(t0) {
       t2 = $[23];
     }
     parsedParams = t2;
-    basePath = "/keystatic";
+    basePath = "/admin";
   }
   const t2 = branch || "";
   let t3;
